### PR TITLE
[terra-arrange] Update docs to use MDX syntax

### PR DESCRIPTION
**Pull request by pranav300**
### Summary
Updated base docs to use MDX syntax.

### CODE REVIEW

File: packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.1.doc.mdx [hunk 1]
```diff
@@ -1,3 +1,14 @@
+import { Badge } from '../../../../package.json?dev-site-package';
```
Comment (mjhenkes) [inline packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.1.doc.mdx]: This import style should work and you should be able to change the rest of the imports this way as well.
```suggestion
import { Badge } from 'terra-arrange/package.json?dev-site-package';
```

File: packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.1.doc.mdx [hunk 2]
```diff
@@ -1,3 +1,14 @@
+import { Badge } from 'terra-arrange/package.json?dev-site-package';
+
+import ArrangeAlignAllContainers from '../example/ArrangeAlignAllContainers?dev-site-example';
+import ArrangeAlignFill from '../example/ArrangeAlignFill?dev-site-example';
+import ArrangeAlignFitEnd from '../example/ArrangeAlignFitEnd?dev-site-example';
+import ArrangeAlignFitStart from '../example/ArrangeAlignFitStart?dev-site-example';
+
+import ArrangePropsTable from '../../../../src/Arrange?dev-site-props-table';
```
Comment (mjhenkes) [inline packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.1.doc.mdx]:
```suggestion
import ArrangePropsTable from 'terra-arrange/src/Arrange?dev-site-props-table';
```

File: packages/terra-arrange/src/terra-dev-site/doc/arrange/ChangeLog.3.doc.mdx [hunk 3]
```diff
@@ -0,0 +1,7 @@
+import { Badge } from 'terra-arrange/package.json?dev-site-package';
+
+import ChangeLog from '../../../../CHANGELOG.md';
```
Comment (mjhenkes) [inline packages/terra-arrange/src/terra-dev-site/doc/arrange/ChangeLog.3.doc.mdx]:
```suggestion
import ChangeLog from 'terra-arrange/CHANGELOG.md';
```

Comment (pranav300) [reply packages/terra-arrange/src/terra-dev-site/doc/arrange/ChangeLog.3.doc.mdx]: Updated here https://github.com/cerner/terra-core/pull/2809/commits/decc9c0ebb0963813d004902c69f4ff662ef2559.

File: packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.1.doc.mdx (hunk 2 above)
Comment (pranav300) [reply packages/terra-arrange/src/terra-dev-site/doc/arrange/Arrange.1.doc.mdx]: Updated here https://github.com/cerner/terra-core/pull/2809/commits/decc9c0ebb0963813d004902c69f4ff662ef2559.

**Pull request merged**